sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -324,20 +324,6 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
       | FROM src LIMIT 1
     """.stripMargin)
 
-  createQueryTest("Date comparison test 2",
-    "SELECT CAST(CAST(0 AS timestamp) AS date) > CAST(0 AS timestamp) FROM src LIMIT 1")
```

Comment:
@adrian-wang at the time of [this fix](https://issues.apache.org/jira/browse/SPARK-6785), Spark was using Hive v0.13, which suffered from the same bug ([HIVE-10178](https://issues.apache.org/jira/browse/HIVE-10178)) and this test case did no longer test for equivalent behavior between Hive and Spark. Please see https://github.com/apache/spark/pull/6242#issuecomment-103670515 (and following) for the discussion about removing this test case.
Since Spark is now on Hive 1.2.1 and [HIVE-10178](https://issues.apache.org/jira/browse/HIVE-10178) was fixed in Hive 1.2, we could consider re-introducing this test case to the `HiveQuerySuite` (@marmbrus ?)